PyInstaller/building/utils.py

```diff
@@ -473,3 +475,93 @@ def format_binaries_and_datas(binaries_or_datas, workingdir=None):
                                 os.path.normpath(src_file)))
 
     return toc_datas
+
+
+def _load_code(modname, filename):
```

Comment:
> You tell me - you wrote it ;).

Argl. I don't remember any more, why. Maybe I wanted to use the same systematic and Python standard procedures. Looks like we should but this on the *long* cleanup list.